Reserved domains off

cctldru opened the pull request:
Description of Organization
====
Coordination Center for TLD RU (CC for TLD RU) is administrator of national top level domains .RU and .РФ (national registry).
Organization Website: https://cctld.ru

Reason of this Pull Request
====

1) CC for TLD RU changed the rules and there are no Reserved domain names at Top level domain .RU at this time. 

Domains 

ac.ru
edu.ru
gov.ru
int.ru
mil.ru
test.ru

is the regular domain names now.

2) New URL for Term and Conditions in section xn--p1ai

sleevi commented:
Thanks! Based on the e-mail addressed provided, I'm guessing you're a representative of the registry operator? We'll work to confirm if you want to go forward, but I do want to highlight that removing these entries would pose serious risk and harm to anyone still in possession of domain registrations within this namespace.

Can you confirm that there are no more registered domains in the `edu.ru` namespace? Does ccTLDru operate `edu.ru` on a registrar basis (i.e. not reserved by the registry, but still permitted for registrations?)

For example, is this PR is accepted, `pnu.edu.ru` would be able to set cookies and other security policy, including possibly compromising, `edem.edu.ru` and `molbiol.edu.ru`. I don't think that's quite what you intended, though?

cctldru commented:
Hi!

I confirm my to authority to present the Coordination Center for TLD .RU and .РФ (.xn--p1ai) (ccTLDru). My name George Georgievsky -- Head of department of work with Registrars and Registrants at .RU and .РФ.

ccTLDru is the Registry and never was operator (registrant or registrar) of _reserved_domain_names_ for special use.

ccTLDru canceled type of domain names for special use (reserved domains) 01 September 2018. But that domain names (ac.ru, edu.ru, gov.ru, int.ru, mil.ru, test.ru) still registered and have his owners (registrants).

My request is changing up information about reserved domains to actual.

The edu.ru domain is a regular domain name in .ru now. 

Third level domains in edu.ru supporting by registrant of edu.ru: Federal state autonomous educational establishment of additional professional education "Center of Realization of State Educational Policy and Informational Technologies" (CRSEPIT) - who operating edu.ru namespace == Registry for third level domains in edu.ru. 

Changing requests for edu.ru and third level domains in edu.ru must send CRSEPIT.
I confirm that Alexander Kozlov <abk@informika.ru> is the response person in edu.ru.

CRSEPIT have some troubles now with Global Sign CA and can not to receive Wildcard SSL for edu.ru with edu.ru string in Public Suffix List.

sleevi commented:
Thanks!

To try and make sure I understand the situation correctly, to figure out next steps:
Prior to 2018-09-01, the domains `ac.ru`, `edu.ru`, `gov.ru`, `int.ru`, `mil.ru`, `test.ru` were specially reserved by Registry policy, delegating authority out to appropriate entities.
On/After 2018-09-01, these domains are no longer special-reserved by policy. They are, however, still registered domains, effectively transitioned to the prior authorities, and many have third-level domain registrations underneath them.

It sounds like we should look at moving these domains from the `ICANN` section (set aside for policy) to the `PRIVATE` section. However, I do want to stress: the proposed change will negatively affect the security of any third-level domains within that namespace, and thus may not be a desirable decision to make. Is this something you'd be willing to bring to the attention of each of these registrants, and see if we can get them engaged on this Pull Request as well? I really don't want to be the one responsible for, e.g., exposing `mil.ru` to a host of new attacks 😬

cctldru commented:
You correctly understand the situation in the first paragraph.

These domains definitely should not be in the ICANN section, and I agree that they need to be moved to the PRIVATE section. But if I understand the PSL rules correctly: should the registrants of these domains send the request of changes themselves?

And it seems to me that transferring to the PRIVATE section will not fix the problem with getting wildcard SSL for CRSEPIT

ccTLDru for its part will inform registrants of these domains about changes in PSL and possible consequences and how they can submit amendments to the PCL.

cctldru commented:
We have a new information from edu.ru registrant: transferring to the PRIVATE section will give them possibility to get wildcard SSL at GlobalSign. You can move 
ac.ru
edu.ru
gov.ru
int.ru
mil.ru
test.ru 

to private section.

cctldru commented:
Follow up, please.

cctldru commented:
New pull request:

Changed comments for domain
ru and domain рф.

Domains:
ac.ru
edu.ru
gov.ru
int.ru
mil.ru
test.ru

moved to PRIVATE DOMAINS section

cctldru commented:
Follow up, please and validate new pull request!

sleevi commented:
Thanks. I'm going to go ahead and merge this. Individual owners for those domains could be separated out, but for situations like this (where the registry used to reserve them), I have no objections to listing the registry as the (old) contact, to preserve the behaviour.
